src/algebra/big_operators/basic.lean

```diff
@@ -1649,8 +1649,78 @@ end list
 
 namespace multiset
 
+lemma disjoint_list_sum_left {a : multiset α} {l : list (multiset α)} :
+  (∀ b ∈ l, multiset.disjoint b a) ↔ multiset.disjoint l.sum a :=
```

Comment:
In my opinion all these lemmas are now backwards! Can you flip the direction of the iffs?
```suggestion
  multiset.disjoint l.sum a ↔ ∀ b ∈ l, multiset.disjoint b a :=
```
(the rule is that generally the iffs should be named after their LHS, and the RHS should be "simpler"; in this case, it eliminates a sum in favor of a binder, which is often considered simpler)

Reply:
Ok, I see. I exchanged the two terms in the iffs of all the lemmas.